clarity_epp.py

```diff
@@ -94,6 +96,8 @@ def export_manual_pipetting(args):
         clarity_epp.export.manual_pipetting.samplesheet_pool_samples(lims, args.process_id, args.output_file)
     elif args.type == 'pool_magnis_pools':
         clarity_epp.export.manual_pipetting.samplesheet_pool_magnis_pools(lims, args.process_id, args.output_file)
+    elif args.type == 'normalization_mix':
+        clarity_epp.export.manual_pipetting.samplesheet_normalization_mix(lims, args.process_id, args.output_file)
```

Comment:
We could probably use the same trick as DxAutomation (using attrgetter). The reason why this was never implemented is because we would like to keep flexibility in the arguments that are passed to the functions. Not all functions expect the same arguments.